src/components/Sections/Topics/TopicPage.js

```diff
@@ -81,33 +82,36 @@ const TopicPage = ({ match, t }) => {
     const breadCrumbIcon = `${isLanguageRtl(language) ? 'left' : 'right'} angle`;
 
     return (
-      <Container className="padded">
-        <Breadcrumb icon={breadCrumbIcon} sections={breadCrumbSections} size="large" />
-        <Divider hidden />
-        <Grid doubling columns={sections.length}>
-          {
-            sections.map(s => {
-              const sectionUnits = getSectionUnits(s);
-              const topicUrl     = getTopicUrl(s, tagPath, language);
-              const sectionCount = getCounts(s);
-
-              return isEmpty(sectionUnits)
-                ? null
-                : (
-                  <Grid.Column key={s}>
-                    <TopN
-                      section={s}
-                      units={sectionUnits}
-                      N={TOP_N_ITEMS}
-                      topicUrl={topicUrl}
-                      sectionCount={sectionCount}
-                    />
-                  </Grid.Column>
-                );
-            })
-          }
-        </Grid>
-      </Container>
+      <div>
```

Comment:
Use <> and </> instead of < div>. <> doesn't create any html, so you will not get problems with CSS, for example